conf/init/nginx_conf_create.py

```diff
@@ -123,6 +123,16 @@ def generate_rate_limiting_config(config):
         static_dir=STATIC_DIR,
     )
 
+def generate_hosted_http_base_config(config):
+    """
+  Generates hosted http base config from the app config
+  """
+    config = config or {}
+    nginx_http_listen_directive_params = config.get("NGINX_HTTP_LISTEN_DIRECTIVE_PARAMS", "default_server")
```

Comment:
Tejas - We should just, instead, add a FEATURE_PROXY_PROTOCOL and if set, add that to the nginx side.